### PR TITLE
Add cortex recording rule for starboard vulnerabilities

### PR DESCRIPTION
**Pull request by stone-z**
This PR:

- adds a rule pushing the number of unique vulnerabilities per severity to cortex

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).
- [ ] Alerting rules must have a comment documenting why it needs to exist.

### CODE REVIEW

**Comment by stone-z** [comment]
Tested on `gaia`

**Comment by stone-z** [comment]
I'd like to get this data in soon so going with a ping, @giantswarm/team-atlas let me know if you have any concerns

File: helm/prometheus-rules/templates/recording-rules/cortex.rules.yml [hunk 1]
```diff
@@ -222,3 +222,6 @@ spec:
     # Falco event counts
     - expr: sum(falco_events{priority=~"0|1|2|3|4|5|6|7"}) by (cluster_type, cluster_id, priority, rule)
       record: aggregation:falco_events
+    # Starboard unique vulnerability counts by severity
+    - expr: count(count by (vulnerability_id, severity) (starboard_exporter_vulnerabilityreport_image_vulnerability)) by (severity)
```
Comment (JosephSalisbury) [inline helm/prometheus-rules/templates/recording-rules/cortex.rules.yml:226]: do you wanna add the cluster id?

Comment (stone-z) [reply helm/prometheus-rules/templates/recording-rules/cortex.rules.yml:226]: Yep, I probably do. Dang it. Thanks Joe, another PR incoming

Comment (JosephSalisbury) [reply helm/prometheus-rules/templates/recording-rules/cortex.rules.yml:226]: well apparently i do not remember how this works. bam!

**Pull request merged**